### DISCUSSION TITLE
Face for topics in draft state?

### DISCUSSION THREAD

**Discussion by sandinmyjoints** [magit/forge#726] [Ideas]
Specifically thinking of Github draft PRs. Would like them to look different from open but non-draft PRs. (I don't know if other forges have a similar notion of draftness.)

Right now, open draft and non-draft Github PRs have the same faces: `forge-topic-pending forge-pullreq-open`

**Comment by tarsius** [comment]
Done in 969b52372a9cdddf23804c999355c72644effeb2.

**Reply by sandinmyjoints** [reply]
Thank you!
